internal/vulnerability/runner.go

```diff
@@ -33,7 +34,13 @@ type VulnerabilityRunner struct {
 }
 
 func NewVulnerabilityRunner(log *logp.Logger) (VulnerabilityRunner, error) {
+	ctx := context.Background() // TODO: carry
 	log.Debug("NewVulnerabilityRunner: New")
+
+	if err := clearTrivyCache(ctx, log); err != nil {
+		log.Warnf("error during runner cache clearing %s", err.Error())
```

Comment:
Why is this a warning and not an error? I mean, this can lead to out of memory, right?

Reply:
Not necessarily; it depends on the error. Since it's not critical to cycle workflow, I used the warning level.